include/openssl/fips_names.h

```diff
@@ -61,6 +61,14 @@ extern "C" {
  */
 # define OSSL_PROV_FIPS_PARAM_TLS1_PRF_EMS_CHECK "tls1-prf-ems-check"
 
+/*
+ * A boolean that determines if the runtime FIPS check for undersized MAC output
+ * is performed.
+ * This is enabled by default.
```

Comment:
enabled??

Reply:
That's what **all** the other Booleans in the file say.